service/main.go

```diff
@@ -1,7 +1,14 @@
 package main
 
-import "github.com/opentdf/platform/service/cmd"
+import (
+	"github.com/opentdf/platform/service/cmd"
+	"github.com/opentdf/platform/service/tracing"
+)
 
 func main() {
+	// Initialize tracer
+	shutdown := tracing.InitTracer()
```

Comment:
Can you file a follow-up issue to make this optional? Even beetter, we would export a tracer API and have applications (e.g. example in this case) inject a Tracer somehow, which would invert the depencency and not require service's go.mod to import otel, etc.